server/src/i_main.cpp

```diff
@@ -114,14 +114,20 @@ int __cdecl main(int argc, char *argv[])
 
     try
     {
-        // Handle ctrl-c, close box, shutdown and logoff events
+        // Handle close box, shutdown and logoff events
         if (!(hEvent = CreateEvent(NULL, FALSE, FALSE, NULL)))
             throw CDoomError("Could not create console control event!\n");
 
         if (!SetConsoleCtrlHandler(ConsoleHandlerRoutine, TRUE))
             throw CDoomError("Could not set console control handler!\n");
 
-        #ifdef _WIN32
```

Comment:
My apologies for my ignorance here, but was the removal of this ifdef intentional?

Reply:
Sorry yes I should have clarified this more in the commit, as on line 94 there is already a #ifdef _WIN32 block protecting the windows main function.